configs/mainnet.yaml

```diff
@@ -157,11 +157,13 @@ WHISK_PROPOSER_SELECTION_GAP: 2
 # EIP7594
 NUMBER_OF_COLUMNS: 128
 MAX_CELLS_IN_EXTENDED_MATRIX: 768
-DATA_COLUMN_SIDECAR_SUBNET_COUNT: 32
+DATA_COLUMN_SIDECAR_SUBNET_COUNT: 128
 MAX_REQUEST_DATA_COLUMN_SIDECARS: 16384
-SAMPLES_PER_SLOT: 8
-CUSTODY_REQUIREMENT: 1
-TARGET_NUMBER_OF_PEERS: 70
+SAMPLES_PER_SLOT: 16
+CUSTODY_REQUIREMENT: 4
+VALIDATOR_CUSTODY_REQUIREMENT: 8
+BALANCE_PER_ADDITIONAL_CUSTODY_SUBNET: 32000000000 # 2**5 * 10**9 (= 32,000,000,000)
+TARGET_NUMBER_OF_PEERS: 100
```

Comment:
Note that most clients don't use this config value, so I guess this is more like a reference / recommendation? 

https://github.com/ethereum/consensus-specs/pull/3766#discussion_r1601222809

Reply:
I have created a PR to remove `TARGET_NUMBER_OF_PEERS` as a config variable
- https://github.com/ethereum/consensus-specs/pull/3852